packages/twenty-front/src/pages/settings/data-model/utils/__tests__/constants.utils.test.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  METADATA_LABEL_VALID_STRING_PATTERN,
+  METADATA_NAME_VALID_STRING_PATTERN,
+  OPTION_VALUE_VALID_STRING_PATTERN,
+} from '~/pages/settings/data-model/utils/constants.utils';
+
+describe('METADATA_NAME_VALID_STRING_PATTERN', () => {
```

Comment:
instead of testing regex, introduce computeMetadataNameFromLabelOrThrow(), computeSelectOptionValueOrThrow

computeMetadataNameFromLabelOrThrow = (string) => transliterateAndValidFormatOrThrow(string, VALID_PATTERN)

rename: formatLabelOrThrow(string, VALID_PATTERN) ==>  transliterateAndValidFormatOrThrow(string, VALID_PATTERN)